raiden/network/proxies/token_network.py

```diff
@@ -227,7 +227,7 @@ def new_netting_channel(
 
         timeout_min = self.settlement_timeout_min()
         timeout_max = self.settlement_timeout_max()
-        invalid_timeout = settle_timeout < timeout_min or settle_timeout > timeout_max
```

Comment:
Going by the variable names, min and max should be allowed. Is there a reason against changing this in the contracts?

Reply:
Actually, I don't think any change is needed. The contract checks for validity inclusively:
https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/data/source/raiden/TokenNetwork.sol#L198-L202

The Raiden code base checks for invalidity exclusively. This matches up perfectly.

Reply:
I try to prove my point in https://github.com/raiden-network/raiden/pull/4774.

Reply:
Oh, you're right. My mistake.